content/applications/hr/employees/new_employee.rst

```diff
@@ -136,14 +148,14 @@ the following information for each skill.
    :align: center
    :alt: A skill form with the information filled out.
```

Comment:
That is because in that particular Runbot, those are the skill types that were added/created to the skill type list. This will appear different based on how the DB was configured. A blank DB won't have any. This pic is from my DB, so that's why the skills you see are different. So this pic is OK in my opinion since it will always appear different for everyone, based on the skill types (which do not come re-configured)